extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -388,13 +394,15 @@
 }
 
 @media (min-width: 952px) {
-  [desktop] .i-amphtml-story-bookend-component {
-    max-width: 50% !important;
+  [desktop] .i-amphtml-story-bookend-component:not(.i-amphtml-story-bookend-heading) {
+    /** Leave some horizontal space between cards. */
+    max-width: calc(50%-12px) !important;
   }
 }
 
 @media (min-width: 1272px) {
-  [desktop] .i-amphtml-story-bookend-component {
-    max-width: 33% !important;
+  [desktop] .i-amphtml-story-bookend-component:not(.i-amphtml-story-bookend-heading) {
+    /** Leave some horizontal space between cards. */
+    max-width: calc(33%-12px) !important;
```

Comment:
Could you please try to match the vertical margin of 24px? That'd give `calc(33% - 16px)`